core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -152,9 +159,30 @@ object BrokerTopicStats extends Logging {
     stats.getAndMaybePut(topic)
   }
 
+  def updateReplicationBytesIn(value: Long) {
+    if (getBrokerAllTopicsStats.replicationBytesInRate.isDefined) {
+      getBrokerAllTopicsStats.replicationBytesInRate.get.mark(value)
```

Comment:
You should use `foreach` instead of `isDefined/get`. Same for the other methods where this applies.